auto_py_to_exe/web/js/i18n.js

```diff
@@ -735,6 +775,7 @@ const translationMap = {
                 cz: 'Pokud chcete přidat soubory do kořenové složky, přidejte tečku (.) do destinace.',
                 gr: 'Έαν θέλετε να εισάγετε τα αρχεία σας στον κεντρικό φάκελο απλά τοποθετήστε μια τελεία (.) στον προορισμό',
                 ar: 'إذا كنت تريد وضع الملفات في الدليل الجذر ، فضع نقطة (.) في الوجهة.',
+                sr: 'Ako želiš da dodaš datoteke u osnovni direktorijum (tzv. root), dodaj tačku (.) u odredištu.,

```

Comment:
Invalid JavaScript - are you sure you ran the application?

> [✔️] Have you run the application to make sure your code runs

Reply:
i totally missed i haven't closed it! it has been fixed now :)
and yes i've ran it and it worked! but i realized i accidentally ran the installed version rather than github version. the application runs properly now

proof: https://imgur.com/a/1QRfvTP